File: 08.typing/typing.cc
#include <iostream>
using namespace std;

int main()
{
   int cats = 2;
   int jobs = 3;
   int reservations = 4;

   int dollars = cats * jobs;
if (reservations == dollars) 
{

    jobs = cats / 5;
  
}
    cout << cats << "\n" << jobs << "\n" << reservations << "\n" << dollars;
   return 0;
}

File: 08.typing/typing.php
<!DOCTYPE HTML>
<html>
    <body>
        <?php 
            $cats = 2;
            $jobs = 3;
            $reservations = 4;
            
            $dollars = $cats * $jobs;
            
            if ($reservations == $dollars) {
                $jobs = $cats / 5;
            }
            
            echo $jobs;
        ?>
    </body>
</html>